frontend/src/pages/data/SubjectView.tsx

```diff
@@ -569,7 +590,7 @@ const SubjectView = observer((props: SubjectViewProps) => {
         totalFormElements++;
         if (formCompleteSoFar) {
             formElements.push(<div key="moco" className="mb-3">
-                <label>Solve For Muscle Activations:</label>
+                <label>Solve For Muscle Activations (Experimental):</label>
```

Comment:
I've actually changed my mind on this one, lets ditch the "Experimental" tag. But could we rename this to "Solve for Muscle Activity"?

Reply:
Done!